controllers/mattermost/mattermost/file_store.go

```diff
@@ -35,7 +41,56 @@ func (r *MattermostReconciler) checkExternalFileStore(mattermost *mmv1beta.Matte
 	return mattermostApp.NewExternalFileStoreInfo(mattermost, *secret)
 }
 
-func (r *MattermostReconciler) checkOperatorManagedMinio(mattermost *mmv1beta.Mattermost, reqLogger logr.Logger) (*mattermostApp.FileStoreInfo, error) {
+func (r *MattermostReconciler) checkLocalFileStore(mattermost *mmv1beta.Mattermost, reqLogger logr.Logger) (mattermostApp.FileStoreConfig, error) {
+	// Default 1Gi volume using default storage class
+	storageSize := mmv1beta.DefaultFilestoreStorageSize
+	if mattermost.Spec.FileStore.Local.StorageSize != "" {
+		storageSize = mattermost.Spec.FileStore.Local.StorageSize
+	}
+	pvc := &corev1.PersistentVolumeClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      mattermost.Name,
+			Namespace: mattermost.Namespace,
+			Labels:    mattermost.MattermostLabels(mattermost.Name),
+		},
+		Spec: corev1.PersistentVolumeClaimSpec{
+			AccessModes: []corev1.PersistentVolumeAccessMode{
+				"ReadWriteOnce",
```

Comment:
I think this AccessMode makes sense, but just wanted to check that it was an intentional choice. We can always extend the local file store configuration to add other options should the need arise, but just wanted to quickly confirm that we think this is the best suited default config.

Reply:
Sorry for the delay on these - I missed notification of comments somehow. 

This was an intentional choice since it seems to be a common default in other apps I use. Thinking about it more, I wonder if this will cause problems with multiple replicas (we only make one PVC per MM instance, not per pod, and pods are not always on the same node). Maybe we want to make the default `ReadWriteMany` to handle that scenario? @gabrieljackson what do you think?

Reply:
Yep, sounds great! I completely agree here and just wanted to check that the choice fits what you had in mind for an initial use-case. Thanks for confirming.